### DISCUSSION TITLE
Github student copilot unavailable

### DISCUSSION THREAD

**Discussion by yatour** [community/community#144644] [GitHub Education]
### Hi everyone,

Why has my student status been revoked and I am currently unable to access GitHub education. 

It seems that a remote login issue has been detected

**Comment by davevad93** [comment]
Hi @yatour ,

This seems to be an account related issue, in order to get a resolution for this kind of things, it's recommended to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem.
